Tangem/Modules/Main/MultiWalletMainContent/MultiWalletMainContentViewModel.swift

```diff
@@ -353,9 +353,16 @@ private extension MultiWalletMainContentViewModel {
     }
 
     func hideToken(tokenItem: TokenItem, blockchainNetwork: BlockchainNetwork) {
-        // TODO: Add analytics
         let derivation = blockchainNetwork.derivationPath
         userWalletModel.userTokensManager.remove(tokenItem, derivationPath: derivation)
+
+        Analytics.log(
+            event: .buttonRemoveToken,
+            params: [
+                Analytics.ParameterKey.source: Analytics.ParameterValue.main.rawValue,
+                Analytics.ParameterKey.token: tokenItem.currencySymbol,
```

Comment:
Чтобы как в другом месте было

```suggestion
                Analytics.ParameterKey.token: tokenItem.currencySymbol,
                Analytics.ParameterKey.source: Analytics.ParameterValue.main.rawValue,
```